gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -64,13 +65,13 @@ def _load_cadd_raw(cadd_tsv) -> hl.Table:
 
         return ht
 
-    snvs = _load_cadd_raw(
-        "gs://gnomad-qin/v4_annotations/cadd.v1.6.whole_genome_SNVs.tsv.bgz"
+    snvs = hl.read_table(
+        "gs://gcp-public-data--gnomad/resources/grch38/in_silico_predictors/CADD_v1.6_SNVs.ht"
```

Comment:
Does this table still exist here? I can't remember which we removed. However, in general, we should keep the original import around or make a note of how this HT was generated. Without either the note or tsv import, someone who didnt run the code will not know how this HT was created.

Reply:
Yes, it exists, we didn't move the ones uploaded to public bucket yet. I can revert it back but change the paths.